### ISSUE TITLE
Cocoapods installation error

### ISSUE THREAD

**Issue by shmakovigor** [Alamofire/Alamofire#1502]
- `Alamofire (~> 4.0)` required by `Podfile`

None of your spec sources contain a spec satisfying the dependency: `Alamofire (~> 4.0)`.

You have either:
 * out-of-date source repos which you can update with `pod repo update`.
 * mistyped the name or version.
 * not added the source repo that hosts the Podspec to your Podfile.

How can I fix this? Thanks.

**Comment by davidcl96** [comment]
Same problem here.

**Comment by thebluepotato** [comment]
NB : this is not a bug, more like a question that could be asked on Stack Overflow, tagged with `alamofire`. Anyhow, that's an easy answer : how could Cocoapods get Alamofire 4.0 if it hasn't been released yet? It's still beta so even though the podspec says 4.0.0, there's no 4.0.0 tag yet!
TL;DR : write `pod 'Alamofire', :git => 'https://github.com/Alamofire/Alamofire.git', :branch => 'master'`

**Comment by davidcl96** [comment]
@thebluepotato Thank you for the explanations.

**Comment by cnoon** [comment]
Thanks @thebluepotato. 🍻
